aten/src/ATen/core/interned_strings.h

```diff
@@ -20,7 +20,9 @@ namespace c10 {
   _(namespaces, namespaces)        \
   _(prim, Assign)                  \
   _(prim, BroadcastingChunk)       \
+  _(prim, BroadcastSizes)          \
```

Comment:
Maybe after this we should look into having symbolic sizes, this makes it feel like those could make things cleaner

Reply:
While I doubt that SymbolicSizes would actually make things cleaner, they would certainly let us at least prove at some broadcasts won't be happening, which would make improving backward fusion quality much easier.